src/lib/routeParams.ts

```diff
@@ -107,116 +73,19 @@ export const useRouteParams = () => {
       }
     }
 
-    if (names[0] === 'storages' && names[1] == null) {
-      return {
-        name: 'workspaces.create',
-      }
-    }
-
     if (names[0] !== 'storages' || names[1] == null) {
       return {
         name: 'unknown',
       }
     }
     const workspaceId = names[1]
-    if (names[2] == null || names[2].length === 0) {
-      return {
-        name: 'workspaces.notes',
-        workspaceId,
-        folderPathname: '/',
-      }
-    }
-
-    let noteId: string | undefined = undefined
-    if (names[2] === 'notes') {
-      const restNames = names.slice(3)
-      if (restNames[0] == null || restNames[0] === '') {
-        return {
-          name: 'workspaces.notes',
-          workspaceId,
-          folderPathname: '/',
-        }
-      }
-
-      const folderNames = []
-      for (const index in restNames) {
-        const name = restNames[index]
-        if (name === '') {
-          break
-        }
-
-        if (/^note:/.test(name)) {
-          noteId = name
-          break
-        }
-
-        folderNames.push(name)
-      }
-
-      return {
-        name: 'workspaces.notes',
-        workspaceId,
-        folderPathname:
-          folderNames.length === 0 ? '/' : '/' + folderNames.join('/'),
-        noteId,
-      }
-    }
-
-    if (names[2] === 'tags') {
-      return {
-        name: 'workspaces.labels.show',
-        workspaceId,
-        tagName: names[3],
-        noteId: /^note:/.test(names[4]) ? names[4] : undefined,
-      }
-    }
-
-    if (names[2] === 'archive') {
-      return {
-        name: 'workspaces.archive',
-        workspaceId,
-        noteId: /^note:/.test(names[3]) ? names[3] : undefined,
-      }
-    }
-
-    if (names[2] === 'attachments') {
-      return {
-        name: 'workspaces.attachments',
-        workspaceId,
-      }
-    }
-
-    if (names[2] === 'timeline') {
-      return {
-        name: 'workspaces.timeline',
-        workspaceId,
-      }
-    }
-
     return {
-      name: 'unknown',
+      name: 'local',
+      workspaceId,
```

Comment:
should it not be safe to remove workspaceId as there is no local workspace anymore?

Reply:
It is handled by Router. The purpose of this method is for parsing pathname.